example/fields.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     fieldClassname: "avatar",
     prompt: "Pick image",
     reprompt: "Pick different image",
-    helpText: "Looks best at 300px × 300px"
+    helpText: "Looks best at 300px × 300px",
+    defaultValue: "https://cdn.pixabay.com/photo/2017/03/17/11/59/food-2151361_1280.jpg"
```

Comment:
turns out this CDN does not like hot linking. Let's just put an image in the `example/` dir and defaultValue-link to that as a relative URL instead.